Refactoring.Pipelines/JoinedPipes.cs

```diff
@@ -63,7 +64,9 @@ private void _SendIfReady()
         {
             if (_values1.Any() && _values2.Any())
             {
-                _Send(Tuple.Create(_values1.Dequeue(), _values2.Dequeue()));
+                _values1.TryDequeue(out TInput1 value1);
```

Comment:
Can/should this use `out var`?

Reply:
My personal opinion is to use "var" only when the type is obvious (e.g. var something = new MyClass()), but I don't feel strongly about that. Is there a ReSharper settings file for the project that I can use to format the code? Or maybe there is, but I don't know how to apply it.